### ISSUE TITLE
[Course]  Deep Learning: Take machine learning to the next level by Arpan Chakraborty, Vincent Vanhoucke

### ISSUE THREAD

**Issue by joyhuang9473** [number9473/datanmachlearn#11]
# [Course]  Deep Learning: Take machine learning to the next level by Arpan Chakraborty, Vincent Vanhoucke
- Author: Arpan Chakraborty, Vincent Vanhoucke
- Origin: [https://www.udacity.com/course/deep-learning--ud730](https://www.udacity.com/course/deep-learning--ud730)
- Related:
  - [course](https://classroom.udacity.com/courses/ud730/lessons/6370362152/concepts/63798118150923)
  - [lecture page](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/udacity/1_notmnist.ipynb)

Lesson 1: From Machine Learning to Deep Learning
- Understand the historical context and motivation for Deep Learning.
- Set up a basic supervised classification task and train a black box classifier on it.
- Train a logistic classifier “by hand”Optimize a logistic classifier using gradient descent, SGD, Momentum and AdaGrad.

Lesson 2: Deep Neural Networks
- Train a simple deep network.
- Effectively regularize a simple deep network.
- Train a competitive deep network via model exploration and hyperparameter tuning.

Lesson 3: Convolutional Neural Networks
- Train a simple convolutional neural net.
- Explore the design space for convolutional nets.

Lesson 4: Deep Models for Text and Sequences
- Train a text embedding model.
- Train a LSTM model.
